cfme/tests/openstack/test_host_security.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon, toolbar
+from cfme.infrastructure.host import Host
+from cfme.web_ui import InfoBlock
+from cfme.fixtures import pytest_selenium as sel
+from cfme.configure.tasks import is_host_analysis_finished
+from utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'],
+                                         scope='module')
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_host_security(provider, soft_assert):
```

Comment:
This test does not do what it is supposed to do by its name. It says it checks *security* but in fact it just checks if SSA reveals any user and any group, so sorry but this is wrong.